svm/src/precompiles.rs

```diff
@@ -0,0 +1,18 @@
+use solana_sdk::{
+    account::{AccountSharedData, WritableAccount},
+    ed25519_program, native_loader, secp256k1_program,
+};
+
+use crate::LiteSVM;
+
+pub(crate) fn load_precompiles(svm: &mut LiteSVM) {
+    let mut account = AccountSharedData::default();
+    account.set_owner(native_loader::id());
+    account.set_lamports(1);
+    account.set_executable(true);
+
+    svm.set_account(ed25519_program::ID, account.clone().into())
```

Comment:
I looked at `bank.rs` and it checks against the `feature_set` before adding the precompile:
```
        for precompile in get_precompiles() {
            let should_add_precompile = precompile
                .feature
                .as_ref()
                .map(|feature_id| self.feature_set.is_active(feature_id))
                .unwrap_or(false);
            if should_add_precompile {
                self.add_precompile(&precompile.program_id);
            }
        }
```
Can we do the same please? We already have a `feature_set` field on `LiteSVM`

Reply:
Implemented in 4ea8b29f073337632e9b3a2f94e1656e0558bdaf

Reply:
Strange, seems the bank code has a logic error/poor wording. These precompiles have feature_id: None which means always enabled. But the code doesn't enable them. My guess is they're loaded elsewhere so it just works... cool.

a0e53b5add03ff2ce03eb9eb8fc3d2fffeff8c28